sw/device/benchmarks/coremark/meson.build

```diff
@@ -0,0 +1,30 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+if target == 'sim-verilator'
+  iterations = '-DITERATIONS=1'
+else
+  iterations = '-DITERATIONS=100'
+endif
+
+
+custom_target(
+  'coremark',
+  command: make_embedded_target,
+  output: make_embedded_target_outputs,
+  build_by_default: true,
+  input: executable(
+    'coremark',
+    sources: ['./top_earlgrey/core_portme.c',
+              './top_earlgrey/ee_printf.c',
+              coremark_base_files],
+    name_suffix: 'elf',
+    dependencies: [
+      sw_lib_uart,
+      sw_lib_mem,
+      riscv_crt,
+    ],
+    c_args : ['-DPERFORMANCE_RUN=1', iterations, '-DTOTAL_DATA_SIZE=2000', '-DMAIN_HAS_NOARGC=1']
+  ),
+)
```

Comment:
Can you follow the template meson file e.g. in hello_world to have an export target available?